cart.spec

```diff
@@ -67,22 +89,32 @@ PREFIX="%{?_prefix}"
 sed -i -e s/${BUILDROOT//\//\\/}[^\"]\*/${PREFIX//\//\\/}/g %{?buildroot}%{_prefix}/TESTING/.build_vars.*
 mv %{?buildroot}%{_prefix}/lib{,64}
 #mv %{?buildroot}/{usr/,}etc
-mkdir -p %{?buildroot}/%{_prefix}/lib/cart
-cp -al multi-node-test.sh utils %{?buildroot}%{_prefix}/lib/cart/
+mkdir -p %{?buildroot}/%{carthome}
+cp -al multi-node-test.sh utils %{?buildroot}%{carthome}/
 mv %{?buildroot}%{_prefix}/{TESTING,lib/cart/}
-ln %{?buildroot}%{_prefix}/lib/cart/{TESTING/.build_vars,.build_vars-Linux}.sh
+ln %{?buildroot}%{carthome}/{TESTING/.build_vars,.build_vars-Linux}.sh
 
 %files
 %defattr(-, root, root, -)
 %{_bindir}/*
 %{_libdir}/*
-%{_prefix}/lib/cart
```

Comment:
is this not needed to be replaced with %{carthome} ?

Reply:
No.  This got [moved](https://github.com/daos-stack/cart/blob/8c5efae17d49fec226c1935d59f921a949b6a8bf/cart.spec#L109-L111) to the _-tests_ sub-package